### ISSUE TITLE
Request for Artifactory Repo

### ISSUE THREAD

**Issue by wenzowski** [BCDevOps/devops-requests#167]
Following the [devhub instructions](https://developer.gov.bc.ca/Using-Artifactory) to request an Artifactory Repo:
* Requester: `awenzows@idir`
* Team Name: [`CAS`](https://github.com/orgs/bcgov/teams/cas)
* Repo Type: [`Helm Chart`](https://www.jfrog.com/confluence/display/RTF/Helm+Chart+Repositories)
* Repo Location: `local`
* Repo Description: `Helm 3 Charts for OCP`
* Primary OpenShift Project: `wksv3k-tools`

Upon approval, the [@bcgov/cas-helm](https://github.com/bcgov/cas-helm) test project will be deprecated and split to its component helm charts, enabling other teams to re-use CI-tested configurations for airflow, redis, postgres, etc. and enabling cross-team collaboration on OCP configuration templates in addition to collaboration on images.

cc @Maralsotoudehnia @matthieu-foucault @thorwolpert

**Comment by wenzowski** [comment]
In case it's not immediately clear, Helm chart repositories contain only Helm charts and do not contain any Docker images. Helm charts are predominantly yaml, and are gzipped prior to upload to the chart repository. As a result, these files are exceptionally small. The compressed `postgresql` chart, for instance, is only 12kb.

TL;DR the resource usage of this repo will be exceptionally small

**Comment by wenzowski** [comment]
Related:
> we are thrilled to announce that OpenShift 4.3 supports Helm 3 as a Tech Preview feature
--[Red Hat Announcement](https://blog.openshift.com/openshift-4-3-deploy-applications-with-helm-3/)

TL;DR latest stable Helm 3.0.3 is now working well with OCP 3.11 and deep integration is planned for future OCP release

**Comment by Maralsotoudehnia** [comment]
@ShellyXueHan @caggles Would it be possible to get this actioned? If so, what do you need from me/the team to implement?

**Comment by ShellyXueHan** [comment]
@wenzowski  done

**Comment by wenzowski** [comment]
For completeness, the created repository is https://artifacts.developer.gov.bc.ca/artifactory/cas-helm-local/ and can be [administered through the artifactory portal](https://artifacts.developer.gov.bc.ca/artifactory/webapp/#/artifacts/browse/tree/General/cas-helm-local)
